docs/content/integrations.md

```diff
@@ -33,6 +32,53 @@ have a valid provider ID (Audible, ITunes or ISBN).
 2. Go to your Ryot user settings and add the correct details as described in the
    [yank](#yank-integrations) section.
 
+### Komga
+
+!!! warning
+
+      This will only import media that are in progress. An import process will be added in
+      the future
+
+The [Komga](https://komga.org/) integration can sync all media if they
+have a valid metadata provider.
+
+#### Komga side steps
+If you use [Komf](https://github.com/Snd-R/komf) or some similar metadata provider these urls will be
+populated automatically. If you don't use komf youll either need to manually add the manga to your collection
+or you can perform the following steps.
+1. Navigate to the manga
+2. Open the edit tab
+3. Navigate to the Links tab
+4. Create a link named `AniList` or `MyAnimeList` providing the respective url (not case-sensitive)
+
+To retrieve your Cookie youll need to perform the following steps:
+1. Log out of Komga
+2. Log back in while selecting `Remember me`
+3. Press F12
+4. Navigate to the Network tab
+5. Refresh the page
+6. Select any of the urls and look for the `Cookie` Header
+7. Copy the entire cookie it should look something like this `remember-me=REDACTED; SESSION=REDACTED`
+
+If you are using NGINX as your reverse proxy youll probably need to add the following to your configuration
```

Comment:
I do not want third party instructions in the docs. If they need help setting up NPM, they can always open an issue. Please remove this section.